GpxExport/ViewControllers/WorkoutsTableViewController.swift

```diff
@@ -90,6 +90,19 @@ class WorkoutsTableViewController: UITableViewController {
     }
 
     private func exportSelectedWorkouts() {
+        let alert = UIAlertController(title: "What file to generate?", message: "It's recommended to use GPX for now.", preferredStyle: .actionSheet)
+
+        alert.addAction(UIAlertAction(title: "GPX", style: .default, handler: {(_: UIAlertAction!) in
+            self.handleExport("GPX")
```

Comment:
Would it make sense to create something like this

```swift
enum ExportFileType {
    case gpx
    case fit
}
```

Reply:
Yeah, lazy me ...